models/classes/class.QtiTestService.php

```diff
@@ -1571,16 +1571,25 @@ private function createTestIdentifier(core_kernel_classes_Resource $test): strin
         return str_replace('_', '-', Format::sanitizeIdentifier($identifier));
     }
 
-    private function getTestLabel(array $testMetadata): string
+    private function getTestLabel(array $testMetadata, XmlDocument $testDefinition): string
     {
         $labelMetadata = array_filter($testMetadata, function ($metadata) {
             return in_array(RDFS_LABEL, $metadata->getPath());
         });
 
+        if (count($labelMetadata) === 0) {
```

Comment:
Please, if possible, use `empty()` method instead
```suggestion
        if (empty($labelMetadata)) {
```